.infrastructure/7_ssm.tf

```diff
@@ -0,0 +1,38 @@
+resource "random_password" "database_password" {
+  length           = 16
+  special          = true
+  override_special = "!#$%&*()-_=+[]{}<>:?"
+}
+
+resource "aws_ssm_parameter" "db_password" { ### SECRET AWS EVENTUALMENTE DA CREARE A MANO PER NON METTERLA SU REPO PUBBLICO
```

Comment:
Add meaningful comments in English, please

Reply:
yes you are right. This is an obsolete comment which I will now remove